### ISSUE TITLE
NY channels removed from US Locals?

### ISSUE THREAD

**Issue by theadder** [epgshare01/share01#144]
Hello.  Have all NY channels been removed permanently from US Locals?  

ABC.(WABC).New.York,.NY.us
CBS.(WCBS).New.York,.NY.us
FOX.(WNYW).New.York,.NY.us
NBC.(WNBC).New.York,.NY.us

They were there until 3 days ago.  Thanks.

**Comment by boeder8** [comment]
Yeah same issue here. I'm guessing they were removed by mistake, I hope it can be fixed.

**Comment by gelliss** [comment]
I mostly use m3u4u for my epg needs but this was great to fill in what that misses.   US locals especially.  Last week they were there and this week many are missing.

**Comment by boeder8** [comment]
> I mostly use m3u4u for my epg needs but this was great to fill in what that misses. US locals especially. Last week they were there and this week many are missing.

M3U4U's epg is mediocre, Ripper's are great and my go-to. I think the channels were removed by error.

**Comment by gelliss** [comment]
> > I mostly use m3u4u for my epg needs but this was great to fill in what that misses. US locals especially. Last week they were there and this week many are missing.
> 
> M3U4U's epg is mediocre, Ripper's are great and my go-to. I think the channels were removed by error.

I only got into using an IPTV  recently after buying a new google tv.  Then downloading tivimate.   I'm so new I don't even know what you mean by "ripper" unless this file here is that?    M3u4u seems fine for me for mimicking a US cable lineup but does miss things, especially locals.   Plus it's nice to have multiple epg sources.

**Comment by boeder8** [comment]
> > > I mostly use m3u4u for my epg needs but this was great to fill in what that misses. US locals especially. Last week they were there and this week many are missing.
> > 
> > 
> > M3U4U's epg is mediocre, Ripper's are great and my go-to. I think the channels were removed by error.
> 
> I only got into using an IPTV recently after buying a new google tv. Then downloading tivimate. I'm so new I don't even know what you mean by "ripper" unless this file here is that? M3u4u seems fine for me for mimicking a US cable lineup but does miss things, especially locals. Plus it's nice to have multiple epg sources.

Ripper Aka share01 (I call him that lol)

**Comment by gelliss** [comment]
> Ripper Aka share01 (I call him that lol)

Ahh. God ya  :)   Do you (or anyone) know of any other free epg's with US locals?   Just in case?   Like I said, it's always good to have backups.

**Comment by benevenstanciano** [comment]
IPTVOrg has a good EPG too. I use a mix of both.

**Comment by benevenstanciano** [comment]
You can search for example on https://iptv-org.github.io/?q=wnyw to see which EPG source contains which channel.
Links are case-sensitive.

**Comment by Kamiela20100** [comment]
I think he resorted back to the previous file yesterday. Change the URL for USA locals back Locals 2
